neqo-transport/src/connection/mod.rs

```diff
@@ -2255,6 +2256,7 @@ impl Connection {
                 pt,
                 pn,
                 &builder.as_ref()[payload_start..],
+                IpTos::default(), // TODO: set from path
```

Comment:
Want to open a bug for that?

Reply:
It's already part of #1678, not sure this would be useful?